Draft: Use case WG review changes (#178)
codatio/codat-docs: docs/underwriting/setting-up.md

---
title: "Setting up for the demo app"
description: "Prepare your Codat instance and local environment for the underwriting app execution"
---

import Tabs from "@theme/Tabs";
import TabItem from "@theme/TabItem";

### 🚀 In this section, you will...
* Create your Codat account, 
* Enable the Assess product, 
* Configure webhooks, 
* Set up your local environment, and
* Run the demo app.

### <input type="checkbox" unchecked/> Create a Codat account 

We provide a [free account](https://signup.codat.io/) that lets you explore and test our APIs and other products, including Assess. It also comes equipped with a sample company. When you start working on your own underwriting solution, you may want to explore other [plans](https://www.codat.io/plans/) that Codat offers. 

### <input type="checkbox" unchecked/> Get your API key

In the [Developers](https://app.codat.io/developers/api-keys) section of the Codat Portal, copy your API key from the **API key** column. You will need this to run the demo app. Make sure you copy the API key, not the auth header.

### <input type="checkbox" unchecked /> Enable the Assess product 

> In the [Products](https://app.codat.io/settings/products) section of the Codat Portal, find **Assess** in the list of products and toggle the switch to enable it. This also enables the data types required by the product, for example, `balanceSheet` and `profitAndLoss` used by the demo app. 

Reviewer: ```suggestion
In the [Products](https://app.codat.io/settings/products) section of the Codat Portal, find **Assess** in the list of products and enable it. The data types required by this product will be configured automatically - for example, `balanceSheet` and `profitAndLoss` will be enabled, which are used by this demo app. 
```

> In this app, we are using the [categorization](https://docs.codat.io/assess/reports/enhanced-financials/categorize-accounts) feature provided by Assess. It allows you to perform automating decisioning based on the categorized chart of accounts. 

Reviewer: ```suggestion
We will use Assess' [categorization](https://docs.codat.io/assess/reports/enhanced-financials/categorize-accounts) feature. It allows you to perform automating decisioning based on the categorized accounts. 
```

Chart of accounts isn't used any more in our docs right?

### <input type="checkbox" unchecked /> Check your auth flow settings

In the **Auth Flow > Link** [settings](https://app.codat.io/settings/link-settings/data-connections), scroll to _Integration categories_. Ensure _Accounting_ integration category is enabled, and the _Sandbox integrations_ switch is toggled on.

You may want to explore and customize Codat's [auth flow](/auth-flow/customize/customize-link) further as part of working on an underwriting solution of your own.

### <input type="checkbox" unchecked /> Configure ngrok

> We recommend using [ngrok](https://ngrok.com/) to access Codat's webhooks. After installing, go to its root directory. Open the `ngrok.exe` file to launch the ngrok terminal. 

Reviewer: Outline the job to be done first? How will webhooks be used in this context? (just one sentence is enough)

Reviewer: ngrok.exe is just PC.

Then, configure your local machine to receive web traffic on port 5069. The demo app is configured to listen for Codat's webhooks on this port.

   ```bash

>    .\ngrok.exe http 5069

Reviewer: Just PC

Reviewer: Mac instructions

```
brew install ngrok
ngrok http 5069
```
